src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -47,7 +47,10 @@
             " (Example: US for United States, MX for Mexico)";
 	public static final String REGEX_FIPS_HYDRO = "[0-9]{12}";
 	public static final String REGEX_FIPS_HYDRO_MESS = "hydrologic unit must match " + REGEX_FIPS_HYDRO;
-
+	public static final String REGEX_MON_LOC_NUMBER = "[0-9]+";
+	public static final String REGEX_MON_LOC_NUMBER_MESS = "monitoring location number must match " + REGEX_MON_LOC_NUMBER;
+	public static final String REGEX_NATIONAL_AQUIFER_CODE = "[A-Z][0-9]+[A-Z]+";
+	public static final String REGEX_NATIONAL_AQUIFER_CODE_MESS = "national aquifer code must match " + REGEX_NATIONAL_AQUIFER_CODE;
```

Comment:
If these messages are shown to the public, it would be better to explain the requirement ('must be all numerical digits') rather than send the regex string to the user.